Classes/Domain/Validator/CategoryMandatoryValidator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/events2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Events2\Domain\Validator;
+
+use JWeiland\Events2\Configuration\ExtConf;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
+use TYPO3\CMS\Extbase\Persistence\ObjectStorage;
+use TYPO3\CMS\Extbase\Validation\Validator\AbstractValidator;
+
+/**
+ * Validates, if category is a mandatory property, if configured in Extension Settings
+ */
+class CategoryMandatoryValidator extends AbstractValidator
+{
+    /**
+     * @param mixed $value
+     */
+    public function isValid($value)
+    {
+        if (!$value instanceof ObjectStorage) {
```

Comment:
Do we need this check? Because inside Model it is explicitly defined as ObjectStorage right?

Reply:
I like such lines as they help me and PhpStorm for type-hinting.
After that line PhpStorm "knows" that $value is of type ObjectStorage.
So, it will not complain about unknown `count()` method.
I would prefer to keep that line